src/pages/graphql/schema/customer/mutations/confirm-email.md

```diff
@@ -0,0 +1,64 @@
+---
+title: confirmEmail mutation | Commerce Web APIs
+---
+
+# changeCustomerPassword mutation
+
+Use the `confirmEmail` mutation to confirm the email address of a customer.
+
+## Syntax
+
+`mutation: {confirmEmail(input: ConfirmEmailInput!) {CustomerOutput}}`
+
+The following call confirms `roni_cost@example.com` as the email address of the customer by using the confirmation key `abcde` that should have been received by the customer in his / her email inbox.
+
+**Request:**
+
+```graphql
+mutation {
+    confirmEmail(input: {
+        email: "roni_cost@example.com"
+        confirmation_key: "abcde"
+    }) {
+        customer {
+            email
+        }
+    }
+}
+```
+
+**Response:**
+
+```json
+{
+  "confirmEmail": {
+    "customer": {
+      "email": "roni_cost@example.com"
+    }
+  }
+}
+```
+
+## Input attributes
```

Comment:
Delete this section. Since we now have autogenerated reference docs, we no longer document the fields and types.